src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -213,6 +224,25 @@ private boolean setupDependencies() throws IOException {
                 }
             }
         }
+        // Android
+        if (target.startsWith("android")) {
+            Path AndroidSdk = configuration.getAndroidSdkPath();
+            Path AndroidNdk = configuration.getAndroidNdkPath();
+
+            Path libsLocation = AndroidSdk.resolve("tools").resolve("lib").resolve("java11");
```

Comment:
`java11` folder is added in `downloadAdditionalAndroidLibs()`.
Those dependencies are needed since `sdkmanager` has a long standing [issue](https://issuetracker.google.com/issues/67495440) about running on java > 8.